sample_code_submission/boosted_decision_tree.py

```diff
@@ -1,90 +1,74 @@
-import numpy as np
-import pandas as pd
+import os
 from xgboost import XGBClassifier
 from sklearn.preprocessing import StandardScaler
-from sklearn.metrics import accuracy_score, roc_auc_score, mean_squared_error
 import pickle
 
+current_dir = os.path.dirname(__file__)
+
 
 class BoostedDecisionTree:
     """
     This class implements a boosted decision tree classifier.
 
     Attributes:
-        * model (XGBClassifier): The underlying XGBoost classifier model.
-        * scaler (StandardScaler): The scaler used to normalize the input data.
+        model (XGBClassifier): The underlying XGBoost classifier model.
```

Comment:
why did you change this ? its causing issues in MACOS ?
the `* ` was added for sphinx documentation

Reply:
I just took the code from codabench submission. Maybe that was old